ocw/lib/aks.py

```diff
@@ -15,33 +12,12 @@ class AKS(Provider):
     def __new__(cls, vault_namespace):
         if vault_namespace not in AKS.__instances:
             AKS.__instances[vault_namespace] = self = object.__new__(cls)
-            self.__container_mgmt_client = None
```

Comment:
@ilausuch  please check this out . WDYT ?

Reply:
I tested locally, and seems that works without this. Good catch